seedfarmer/commands/_stack_commands.py

```diff
@@ -459,7 +460,7 @@ def deploy_seedkit(
     security_group_ids: Optional[List[str]]
         The Security Group IDs to associate seedkit with (codebuild)
     """
-    session = SessionManager().get_or_create().get_deployment_session(account_id=account_id, region_name=region)
+    session = SessionManager().get_or_create(profile=profile).get_deployment_session(account_id=account_id, region_name=region)
```

Comment:
At this time, the toolchain role has been created as it is a singleton....ONLY the toolchain role can assume the deployment roles ...